src/app/shared/services/dia-backend/series/dia-backend-series-repository.service.ts

```diff
@@ -0,0 +1,52 @@
+// tslint:disable: no-magic-numbers
+import { HttpClient, HttpErrorResponse } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { defer, Subject, throwError } from 'rxjs';
+import { catchError, concatMap } from 'rxjs/operators';
+import { DiaBackendAuthService } from '../auth/dia-backend-auth.service';
+import { NotFoundErrorResponse } from '../errors';
+import { BASE_URL } from '../secret';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class DiaBackendSeriesRepository {
+  private readonly postCapturesUpdated$ = new Subject<{ reason?: string }>();
+
+  constructor(
+    private readonly httpClient: HttpClient,
+    private readonly authService: DiaBackendAuthService
+  ) {}
+
+  readSeries$() {
```

Comment:
Rename to `fetchSeries$` for consistency.

Also, as it does NOT have any argument, this method can be a readonly member. For example:

```suggestion
  readonly fetchSeries$ = defer(() => this.authService.getAuthHeaders()).pipe(
    concatMap(headers =>
    ...
    )
 )
```